README.md

```diff
@@ -1,2 +1,4 @@
 # neptune_test
 Neptune Test Repository
+
+This is to compare the difference between two files.
```

Comment:
@ramanathants Could you please add change this to "This is to compare the difference between **the** two files."?